src/rpc/rawtransaction.cpp

```diff
@@ -284,8 +284,13 @@ UniValue gettxoutproof(const JSONRPCRequest& request)
         pblockindex = mapBlockIndex[hashBlock];
     } else {
         CCoins coins;
-        if (pcoinsTip->GetCoins(oneTxid, coins) && coins.nHeight > 0 && coins.nHeight <= chainActive.Height())
```

Comment:
ha. Yes, you're right. This is the last transaction in the list.

`%s/first/last/g` my original comment